search-services/latest/config/indexing.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Indexing recommendations
+---
+
+When upgrading from a previous Search Services version you should review these indexing considerations:
+
+* [Cross Locale](#cross-locale) Enabling or disabling
+* [Exact term search](#exact-term-search)
+* [Document fingerprints]({% link search-services/latest/config/performance.md %}#disable-document-fingerprints) Enabling or disabling
```

Comment:
Link should be insight-engine/latest/config/performance/#disable-document-fingerprint (not that there is no s at the end)

Reply:
Fixed for both.